js/handlers.js

```diff
@@ -188,11 +188,18 @@ weechat.factory('handlers', ['$rootScope', '$log', 'models', 'plugins', 'notific
                 buffer = new models.Buffer(bufferInfos[i]);
                 models.addBuffer(buffer);
                 // Switch to first buffer on startup
-                if (i === 0) {
+                var shouldResume = bufferResume.shouldResume(buffer);
+                if(shouldResume){
```

Comment:
this assignment should probably be inlined into the condition - `if (bufferResume.shouldResume(buffer)) {`